utils/utils.go

```diff
@@ -58,9 +59,8 @@ func eofKeySequenceText() string {
 func AddSecret(toAdd Secret, keyURI string, keyCheck bool, envelope string) error {
 	stateFile := V2{Filepath: envelope}
 	stateFile.KeyIdentifier = keyURI
-	nvl := NewVersionedLoader(envelope)
 
-	contents, err := nvl.ReadState()
+	contents, err := LoadEnvelope(envelope)
```

Comment:
I like how this pr is cleaning up some verbaige i left behind in refactoring from the pre 1.x to 1.x. :+1: